src/selectedtests/task_mappings/get_task_mappings.py

```diff
@@ -5,21 +5,6 @@
 from pymongo.collection import Collection
 
 
-def _exclude_tasks_below_threshold(task_mapping: dict, threshold: Decimal) -> dict:
-    """
-    Retrieve task mappings associated with a given evergreen project and list of source files.
-
-    :param task_mapping: The task mapping being analyzed.
-    :param threshold: Min threshold desired for flip_count/source_file_seen_count ratio.
-    :return: The task mapping minus the tasks that fell below the desired threshold.
-    """
-    for task in list(task_mapping["tasks"]):
-        if (task["flip_count"] / task_mapping["source_file_seen_count"]) < threshold:
-            task_mapping["tasks"].remove(task)
-
-    return task_mapping
-
-
 def get_correlated_task_mappings(
```

Comment:
Has the test been updated for this method? (I assume the filtering logic tests should be removed?)